java/code/src/com/redhat/rhn/frontend/xmlrpc/system/scap/SystemScapHandler.java

```diff
@@ -156,7 +157,7 @@ public int scheduleXccdfScan(User loggedInUser, List serverIds,
      * @xmlrpc.returntype #param_desc("int", "id", "ID if SCAP action created")
      */
     public int scheduleXccdfScan(User loggedInUser, List serverIds,
-             String xccdfPath, String oscapParams, Date date) {
+             String xccdfPath, String oscapParams, String ovalFiles, Date date) {
```

Comment:
For backward compatibility we need to keep the old signature and create a copy with the new signature

Reply:
Oops, I missed that one. FIxing it :+1: